Volume/View Models/MagazineCell.swift

```diff
@@ -8,23 +8,29 @@
 
 import SDWebImageSwiftUI
 import SwiftUI
+import PDFKit
 
 /// `MagazineCell` displays the images and name of a publication a user is currently following
 struct MagazineCell: View {
 
     let magazine: Magazine
-    // let largeFont: Bool
+    let magazineUrl: URL
 
     var body: some View {
         VStack(alignment: .leading) {
-            // TODO: figure out how to obtain cover image from PDF
-            WebImage(url: URL(string: "https://picsum.photos/200")!)
-                .resizable()
-                .grayBackground()
-                .aspectRatio(contentMode: .fill)
-                .frame(width: 150, height: 220)
-                .clipped()
-                .shadow(color: Color.black.opacity(0.2), radius: 8, x: 4, y: 4)
+            if let pdfDoc = PDFDocument(url: magazineUrl) {
+                PDFKitView(pdfDoc: pdfDoc)
+                    .frame(width: 150, height: 220)
+                    .scaledToFill()
+                    .shadow(color: Color.black.opacity(0.2), radius: 8, x: 4, y: 4)
+                    .disabled(true)
+            } else {
+                PDFKitView(pdfDoc: PDFDocument())
+                    .frame(width: 150, height: 220)
+                    .scaledToFill()
+                    .shadow(color: Color.black.opacity(0.2), radius: 8, x: 4, y: 4)
+                    .disabled(true)
```

Comment:
One example of where constants would be nice cuz it's used twice. You could also wrap the `if...else` in a `Group` and apply the `.frame`...`disabled` modifiers once to the group so you don't have to write it twice.